doc/collections/_pages/content/handbook/style/colons.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Colons
+key_point: A colon indicates that closely-related information follows.
+original_url: https://developers.google.com/style/colons
+original_title: Colons | Google developer documentation style guide
+---
+
+<section id="phrase-preceding-colon" markdown="1">
+
+## Introductory phrase preceding colon
+
+When a colon introduces a list, the text that precedes the colon must be able to
+stand alone as a complete sentence.
+
+### Examples
+
+<p class="example">
+  <span class="compare-better">Recommended:</span> The fields are defined as
+  follows:
+</p>
+<p class="example">
+  <span class="compare-worse">Not recommended:</span> The fields are:
+</p>
+
+</section>
+
+<section id="bold-text-preceding-colon" markdown="1">
+
+## Bold text preceding colon
```

Comment:
This actually goes against what Node.js does for their release changelogs and me just observing everything else that is going on, this might actually need to be changed to _not_ include the colon. 🤔

Refs: https://github.com/nodejs/changelog-maker